ThreadGroup/Controllers/TGRouterAuthViewController.h

```diff
@@ -12,11 +12,13 @@
 @class TGRouterAuthViewController;
 
 @protocol TGRouterAuthViewControllerDelegate <NSObject>
-- (void)routerAuthenticationSuccessful:(TGRouterAuthViewController *)routerAuthenticationView;
 - (void)routerAuthenticationCanceled:(TGRouterAuthViewController *)routerAuthenticationView;
+- (void)okButtonWasPressedInRouterAuthentication:(TGRouterAuthViewController *)routerAuthenticationView;
```

Comment:
I would standardize these delegate method names, i.e

routerAuthenticationViewControllerDidPressOkButton:(TGRouterAuthViewController *)routerAuthenticationView.

I would even update the existing methods, but maybe check with Pat or Lu first.